get-cloudify.py

```diff
@@ -109,20 +110,22 @@
 IS_VIRTUALENV = hasattr(sys, 'real_prefix')
 
 REQUIREMENT_FILE_NAMES = ['dev-requirements.txt', 'requirements.txt']
+
 # TODO: put these in a private storage
-PIP_URL = 'http://repository.cloudifysource.org/org/cloudify3/components/get-pip.py'  # NOQA
-PYCR64_URL = 'http://repository.cloudifysource.org/org/cloudify3/components/pycrypto-2.6.win-amd64-py2.7.exe'  # NOQA
-PYCR32_URL = 'http://repository.cloudifysource.org/org/cloudify3/components/pycrypto-2.6.win32-py2.7.exe'  # NOQA
+repo = 'http://repository.cloudifysource.org/org/cloudify3/components'
+PIP_URL = '{repo}/get-pip.py'.format(repo=repo)
+PYCR64_URL = '{repo}/pycrypto-2.6.win-amd64-py2.7.exe'.format(repo=repo)
+PYCR32_URL = '{repo}/pycrypto-2.6.win32-py2.7.exe'.format(repo=repo)
 
 PLATFORM = sys.platform
 IS_WIN = (PLATFORM == 'win32')
```

Comment:
I suggest we replace with os.name == 'nt' instead.. which is the standard way of checking this.